content/tutorials/getting-started.md

```diff
@@ -23,7 +23,96 @@ It should only be used for testing purposes.
+   |----------------|----------|----------------------------------------------------------------------------------------------|
+   | Linux          | amd64    | [linux-amd64](https://github.com/minio/kes/releases/latest/download/kes-linux-amd64)         |
+   | Linux          | arm64    | [linux-arm64](https://github.com/minio/kes/releases/latest/download/kes-linux-arm64)         |
+   | Linux          | ppc64le  | [linux-ppc64le](https://github.com/minio/kes/releases/latest/download/kes-linux-ppc64le)     |
+   | Linux          | s390x    | [linux-s390x](https://github.com/minio/kes/releases/latest/download/kes-linux-s390x)         |
+   | Apple (M1, M2) | arm64    | [darwin-arm64](https://github.com/minio/kes/releases/latest/download/kes-darwin-arm64)       |
+   | Apple (Intel)  | amd64    | [darwin-amd64](https://github.com/minio/kes/releases/latest/download/kes-darwin-amd64)       |
+   | Windows       | amd64    | [windows-amd64](https://github.com/minio/kes/releases/latest/download/kes-windows-amd64.exe) |
+
+   - Optionally, verify the binary using [minisign](https://jedisct1.github.io/minisign/) by downloading the corresponding [`.minisign` signature file](https://github.com/minio/kes/releases/latest).
+     Then fun the following command, replacing `<OS>` and `<ARCH>` with the values for your system:
+
+     ```shell {.copy}
+     minisign -Vm kes-<OS>-<ARCH> -P RWTx5Zr1tiHQLwG9keckT0c45M3AGeHD6IvimQHpyRywVWGbP1aVSGav
+     ```
+
+   - Mark the downloaded file as executable if necessary for your OS.
+
+     For example, run `chmod +x <path/to/file>`.
+   - Optionally, move the file to the desired run location or to a folder in your system's path or add the file's location to the system path.
+
+     For example, if you put the binary in `$HOME/minio-binaries`, run the following:
+
+     ```shell {.copy}
+     export PATH=$PATH:@HOME/minio-binaries/
+     ```
+
+   **Important:** Invoke the Windows executable file from the terminal, PowerShell, or Command Prompt.
+   You cannot double click the file from the Windows graphical user interface.
+
+   {{< /tab >}}
+
+   {{< tab "Docker" >}} 
+   Pull the release with the following command:
+
+   ```shell {.copy}
+   docker pull minio/kes
+   ``` 
+
+   {{< /tab >}}
+
+   {{< tab "Homebrew" >}} 
+
+   To install the KES binary with [Homebrew](https://brew.sh), run the following command:
+
+   ```shell {.copy}
+   brew install minio/stable/kes
+   ```
+
+   {{< /tab >}}
+
+   {{< tab "Source" >}} 
+
+   You can download and install the binary with your Go toolchain:
+
+   ```shell {.copy}
+   go install github.com/minio/kes/cmd/kes@latest
+   ```
+
+   {{< /tab >}}
+
+   {{< /tabs >}}
+
+   Confirm command availability by running the following command from your prompt or terminal:
+
+   ```shell {.copy}
+   kes --help
+   ```
+
+   You may need to adjust the command for your OS structure and PATH, such as with `./kes --help`.
```

Comment:
I think the recommendation here is to read the `--help` output to learn what to do? Perhaps something like `see /.kes --help for more information`